app/components/login_button_component.rb

```diff
@@ -0,0 +1,26 @@
+class LoginButtonComponent < ButtonComponent
+  attr_reader :action, :color, :logo_path, :tag_options
+
+  def initialize(color: "light blue", **tag_options)
```

Comment:
my expectation is that the components typically use symbol slug names (ex `:light_blue` here), WDYT of doing the mapping from the human-friendly names in the `preview.rb` file?